dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Grid/LoadFlow.mo

```diff
@@ -0,0 +1,807 @@
+within Dynawo.Examples.Nordic.Grid;
```

Comment:
I think this could be put in TestCases, what do you think ?

Reply:
I agree with you. Is the LoadFlow an INIT model ?

Reply:
Moved to TestCases